src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -154,6 +154,7 @@ public sealed partial class EventHandlerList : System.IDisposable
         public void Dispose() { }
         public void RemoveHandler(object key, System.Delegate? value) { }
     }
+    [System.ComponentModel.TypeConverterAttribute("System.ComponentModel.ComponentConverter, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
A quick search through corefx suggests we have a bunch of TypeConverter attributes that aren't in the refs.  Should they be?  Or is this not needed?

Reply:
It'd simplify things if the typeconverters didn't point to typeforwarders

Reply:
> Should they be?

Yes, WPF depends on it.  Can you point out a place where it's missing?  APICompat should check for this at the type level and I don't believe we have much use at member level today.

> It'd simplify things if the typeconverters didn't point to typeforwarders

This was an intentional decision.  Can you let me know what is simplified by avoiding typeforwards?  Would it be sufficient to have a lower version?

Reply:
@stephentoub To be clear, I specifically added the attribute check to APICompat to preserve TypeConverter attribute, so I don't expect it to be missing.  If you found anywhere we missed it in the reference assembly that'd be a bug.

@marek-safar when we're using type name types upstack we've chosen to use the desktop identity since it is something that is stable and doesn't change.  That way folks aren't tempted to version these strings with every release.  I'd be open to hearing an argument why it's better not to do this.

Reply:
> so I don't expect it to be missing. If you found anywhere we missed it in the reference assembly that'd be a bug.

From what I can tell some are.  For example, DataColumn.DefaultValue has one in src:
https://github.com/dotnet/corefx/blob/279292804fa4f6cb856577e59246ac934a2ef9e5/src/System.Data.Common/src/System/Data/DataColumn.cs#L641-L642
but not in the ref:
https://github.com/dotnet/corefx/blob/279292804fa4f6cb856577e59246ac934a2ef9e5/src/System.Data.Common/ref/System.Data.Common.cs#L125
or as another example SqlClient has a couple in src:
https://github.com/dotnet/corefx/blob/279292804fa4f6cb856577e59246ac934a2ef9e5/src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnectionStringBuilder.cs#L387-L388
https://github.com/dotnet/corefx/blob/279292804fa4f6cb856577e59246ac934a2ef9e5/src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameter.cs#L74-L75
and "TypeConverter" doesn't appear anywhere in the ref:
https://github.com/dotnet/corefx/blob/17382def0f680653870c31af1acf086ac41dcc0b/src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

Reply:
Yep, those are instance members not class level.  We don't yet run APICompat for those which is the reason for the discrepancy, haven't had folks ask for them, but we can fix them.  I believe XAML would use on them if it saw them.

Reply:
> I'd be open to hearing an argument why it's better not to do this.

It complicates the linking story. The linker will have to either rewrite the string (basically fixing the code) or there will have to be an extra assembly to proxy to the right type which looks to me for WASM like scenarios as a not optimal solution.

Reply:
That's unfortunate. I didn't know it's used all the place.

Reply:
@stephentoub I added the member attribute diffing to APICompat and will submit a seperate PR for that which brings the repo in sync.  I suspect it will require some conversation so I don't want to block this PR on it.